ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/disconnected/ovn/cucushift-installer-rehearse-baremetalds-ipi-disconnected-ovn-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: cucushift-installer-rehearse-baremetalds-ipi-disconnected-ovn
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-baremetalds-ipi-disconnected-provision
+    post:
+    - chain: cucushift-installer-rehearse-baremetalds-ipi-disconnected-deprovision
```

Comment:
seem like the chain make no difference with `cucushift-installer-rehearse-baremetalds-ipi-deprovision`, I think you can directly call `cucushift-installer-rehearse-baremetalds-ipi-deprovision` here.